addons/website_sale/models/sale_order.py

```diff
@@ -564,9 +564,9 @@ def _message_post_after_hook(self, message, msg_vals):
             self.cart_recovery_email_sent = True
         return super()._message_post_after_hook(message, msg_vals)
 
-    def _notify_get_recipients_groups(self, message, model_description, msg_vals=None):
-        """ In case of cart recovery email, update link to redirect directly
-        to the cart (like ``mail_template_sale_cart_recovery`` template). """
+    def _notify_get_recipients_groups(self, message, model_description, msg_vals=False):
+        # In case of cart recovery email, update link to redirect directly
+        # to the cart (like ``mail_template_sale_cart_recovery`` template). """
```

Comment:
```suggestion
        # to the cart (like ``mail_template_sale_cart_recovery`` template).
```

:eyes:
I guess you remove those so that you see the main method docstring on hover in the IDE ?

Reply:
Indeed ^^